src/main/kotlin/com/autonomousapps/internal/utils/moshi.kt

```diff
@@ -10,10 +10,14 @@ import com.squareup.moshi.*
 import com.squareup.moshi.Types.newParameterizedType
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import dev.zacsweers.moshix.sealed.reflect.MetadataMoshiSealedJsonAdapterFactory
+import okio.BufferedSource
 import okio.buffer
 import okio.sink
 import java.io.File
 
+const val noJsonIndent = ""
+const val prettyJsonIndent = "  "
```

Comment:
can these be private?

Reply:
No, getting `Error:(98, 46) Public-API inline function cannot access non-public-API 'private const val prettyJsonIndent: String defined in com.autonomousapps.internal.utils in file moshi.kt'` for `inline fun <reified T> T.toPrettyString(withNulls: Boolean = false): String` and others then.

Reply:
what about `internal`, then?

Oh hm, I note that all the functions here are public. I feel like I did that for a reason, but I don't remember now. I guess it's fine, I can take a look myself separately.

Reply:
Same issue `Public-API inline function cannot access non-public-API 'internal const val prettyJsonIndent: String defined in com.autonomousapps.internal.utils in file moshi.kt'`. I'm not too familiar with Kotlin and the extension functions, open for suggestions on how to change it :)

Reply:
No worries, I'll take a look at this myself! Thanks.